react-events/my-app/src/Button.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  label: string;
+  buttonClicked: () => void;
```

Comment:
Remember to follow the naming convention for event handler props and start them with `on`; that is, `onButtonClicked`.